rice-middleware/impl/src/main/resources/org/kuali/rice/kew/impl/document/search/DocumentSearchCriteriaBo.xml

```diff
@@ -69,38 +69,38 @@
             </list>
           </property>
         </bean>
-        <bean parent="RelationshipDefinition">
-          <property name="objectAttributeName" value="initiatorPerson"/>
-          <property name="supportAttributes">
-            <list>
-              <bean parent="SupportAttributeDefinition" p:identifier="true" p:sourceName="initiatorPrincipalName" p:targetName="principalName"/>
-            </list>
-          </property>
-        </bean>
-        <bean parent="RelationshipDefinition">
-          <property name="objectAttributeName" value="approverPerson"/>
-          <property name="supportAttributes">
-            <list>
-              <bean parent="SupportAttributeDefinition" p:identifier="true" p:sourceName="approverPrincipalName" p:targetName="principalName"/>
-            </list>
-          </property>
-        </bean>
-        <bean parent="RelationshipDefinition">
-          <property name="objectAttributeName" value="viewerPerson"/>
-          <property name="supportAttributes">
-            <list>
-              <bean parent="SupportAttributeDefinition" p:identifier="true" p:sourceName="viewerPrincipalName" p:targetName="principalName"/>
-            </list>
-          </property>
-        </bean>
-        <bean parent="RelationshipDefinition">
-          <property name="objectAttributeName" value="groupViewer"/>
-          <property name="supportAttributes">
-            <list>
-              <bean parent="SupportAttributeDefinition" p:identifier="true" p:sourceName="groupViewerId" p:targetName="id"/>
-            </list>
-          </property>
-        </bean>
+<!--        <bean parent="RelationshipDefinition">-->
```

Comment:
is there a benefit to leaving this commented out, or should it just be completely removed?

Reply:
I decided to leave them in so that we can just uncomment them when they are pulled into KFS to get that functionality back.